.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+
+### Reporting New Bugs :bug:
+
+If you experience a bug and there is no discussion or open issue about it, please considering starting a new discussion topic or opening a new issue to report it. It is extremely important that you try to gather as much information about the issue as possible. Screenshots, files, plots and commands are useful and help developers to address the issue. The following is a list of possible things to include when opening a new issue:
+
+* Use a clear and descriptive title when opening the issue. This help developers to identify the problem more easily.
+* Provide a description of the steps you have taken to reproduce the problem with as many details as you can. Don't just explain what you did, but include the information on how you did it.
+* Describe what behavior you observed after following the steps and what was the behavior you were expecting to observe. Also provide the reason why you expected to observe a different outcome.
+* Include screenshots, files, plots and commands that you used when you experienced the problem as means to illustrate the issue.
+* In addition, please provide the description of the Modelica-compliant software that was used when you experienced the problem. Some issues can be tool-related and this allow us to help their developers to improve their tool.
+
+
+### Improving Documentation :memo:
+
+The documentation of the OpenIPSL is located within the Modelica package. This documentation is added via annotations in the Modelica models and there are even dedicated files inside the library that are basically composed of annotations.
+In addition to that, markdown files are present in many folders of the library, providing description to the many aspects of this library.
+
+Indeed, the main goal of the documentation is to provide information on the development conventions adopted for this project, and provide a guide to the contributors of the project. If you want to contribute to building a better documentation, you can first start by learning about [Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/about-writing-and-formatting-on-github) and about [Modelica Annotations](https://mbe.modelica.university/behavior/equations/annotations/) if you are not familiar with these concepts yet. Please, visit the [Issues List](https://github.com/OpenIPSL/OpenIPSL/issues) since there are Documentation-related issues that can help you better understand what is needed by OpenIPSL.
```

Comment:
```suggestion
Indeed, the main goal of the documentation is to provide information on the development conventions adopted for this project, and provide a guide to the contributors of the project. If you want to contribute to building a better documentation, you can start by learning about [Modelica Annotations](https://mbe.modelica.university/behavior/equations/annotations/) if you are not familiar with this concept yet. Please, visit the [Issues List](https://github.com/OpenIPSL/OpenIPSL/issues) since there are Documentation-related issues that can help you better understand what is needed by OpenIPSL.
```